packages/forms/src/directives/validators.ts

```diff
@@ -140,6 +140,11 @@ abstract class AbstractValidatorDirective implements Validator {
   registerOnValidatorChange(fn: () => void): void {
     this._onChange = fn;
   }
+
+  // TODO: add docs as needed
+  enabled(): boolean {
+    return this.inputName != null /* both `null` and `undefined` */;
```

Comment:
I think this change is incorrect since `this.inputName` would contain the name of the input, not its value. The change should look like this:
```suggestion
    return this[this.inputName] != null /* both `null` and `undefined` */;
```

Reply:
yes you are correct. for now I am getting error
`Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 'AbstractValidatorDirective'.
  No index signature with a parameter of type 'string' was found on type 'AbstractValidatorDirective'` will update here once it's resolved

Reply:
I was thinking what if I capture value of element in handleChanges and use of it like below, let me know your thoughts @AndrewKushnir 

```
handleChanges(changes: SimpleChanges): void {
    if (this.inputName in changes) {
      this._inputValue = this.normalizeInput(changes[this.inputName].currentValue);
      this._validator = this.createValidator(this._inputValue);
      if (this._onChange) {
        this._onChange();
      }
    }
  }
  
  
  ...
  // TODO: add docs as needed.
  enabled(): any {
    return this._inputValue != null /* both `null` and `undefined` */;
  }
```

Reply:
@iRealNirmal I think you can use a type cast there:

```
return (this as unknown as {[key: string]: unknown})[this.inputName] != null /* both `null` and `undefined` */;
```

Reply:
yes @AndrewKushnir it helped, I will do the required change. Thanks for helping in it.